services/minespace-web/src/components/pages/Project/InformationRequirementsTableStepForm.tsx

```diff
@@ -158,26 +158,24 @@ const StepForms = ({
                   project?.information_requirements_table?.irt_guid
                 )}
               >
```

Comment:
SonarCloud is making 2 complaints about this file:
1. remove the import for AuthorizationWrapper
2. rename stepFormsProps --> StepFormProps

May as well fix them!